src/view/screens/Settings/index.tsx

```diff
@@ -647,6 +647,31 @@ export function SettingsScreen({}: Props) {
             <Trans>External Media Preferences</Trans>
           </Text>
         </TouchableOpacity>
+        <TouchableOpacity
+          testID="linkToChatSettingsBtn"
+          style={[
+            styles.linkCard,
+            pal.view,
+            isSwitchingAccounts && styles.dimmed,
+          ]}
+          onPress={
+            isSwitchingAccounts
+              ? undefined
+              : () => navigation.navigate('MessagesSettings')
+          }
+          accessibilityRole="button"
+          accessibilityLabel={_(msg`Chat settings`)}
+          accessibilityHint={_(msg`Opens chat settings`)}>
+          <View style={[styles.iconContainer, pal.btn]}>
+            <FontAwesomeIcon
+              icon={['far', 'comment-dots']}
+              style={pal.text as FontAwesomeIconStyle}
+            />
+          </View>
+          <Text type="lg" style={pal.text}>
+            <Trans>Chat Settings</Trans>
```

Comment:
```suggestion
            <Trans>Chat Preferences</Trans>
```

We use "Preferences" everywhere else here, but fwiw we _are_ using "Settings" on the actual settings screen itself...

Reply:
Don't feel strongly, we can change this whenever.

Reply:
I think Settings here, just so it's the same when you click through. Don't feel strongly either tho

Reply:
Yea I realized that after going into the screen. Another option might be to just drop the "Settings" all together. You're in "Settings" already so you know it takes you to more settings.

Again though, that doesn't follow the other behavior we have lol...probably just something to revisit when we rework this screen hopefully soon.

Reply:
can't wait to burn it all down fr